README.md

```diff
@@ -132,6 +132,130 @@ PrestoGateway records history of recent queries and displays links to check quer
+        "hardCpuLimit": null, \
+        "parent": null, \
+        "environment": "test" \
+    }'
+```
+
+### Get existing resource group(s)
+If no resourceGroupId (type long) is specified, then all existing resource groups are fetched. 
+```$xslt
+curl -X GET http://localhost:8080/presto/resourcegroup/read/{INSERT_ID_HERE}
+```
+
+### Update a resource group
+Specify all fields in the body, which will overwrite the existing resource group's properties.
+```$xslt
+curl -X POST http://localhost:8080/presto/resourcegroup/update \
+ -d '{  "resourceGroupId": 1, \
+        "name": "resourcegroup_updated", \
+        "softMemoryLimit": "80%", \
+        "maxQueued": 50, \
+        "softConcurrencyLimit": 40, \
+        "hardConcurrencyLimit": 60, \
+        "environment": "test", \
+        "schedulingPolicy": null, \
+        "schedulingWeight": null, \
+        "jmxExport": null, \
+        "softCpuLimit": null, \
+        "hardCpuLimit": null, \
+        "parent": null, \
+        "environment": "test" \
+    }'
+```
+
+### Delete a resource group
+To delete a resource group, specify the corresponding resourceGroupId (type long).
+```$xslt
+curl -X POST http://localhost:8080/presto/resourcegroup/delete/{INSERT_ID_HERE}
+```
+
+### Add a selector
+To add a single selector, specify all relevant fields in the body.
+```$xslt
+curl -X POST http://localhost:8080/presto/selector/create \
+ -d '{  "resourceGroupId": 1, \
+        "priority": 1, \
+        "userRegex": "selector1", \
+        "sourceRegex": "resourcegroup1", \
+        "queryType": "insert" \
+     }'
+```
+
+### Get existing selectors(s)
+If no resourceGroupId (type long) is specified, then all existing selectors are fetched. 
+```$xslt
+curl -X GET http://localhost:8080/presto/selector/read/{INSERT_ID_HERE}
+```
+
+### Update a selector
+Specify all fields in the body, which will overwrite the existing selector's properties.
+```$xslt
+curl -X POST http://localhost:8080/presto/selector/update \
+ -d '{  "resourceGroupId": 1, \
+        "priority": 2, \
+        "userRegex": "selector_updated", \
+        "sourceRegex": "resourcegroup1", \
+        "queryType": "insert" \
+     }'
+```
+
+### Delete a selector
+To delete a selector, specify the corresponding resourceGroupId (type long).
+```$xslt
+curl -X POST http://localhost:8080/presto/selector/delete/{INSERT_ID_HERE}
+```
+
+### Add a global property
+To add a single global property, specify all relevant fields in the body.
+```$xslt
+curl -X POST http://localhost:8080/presto/globalproperty/create \
+ -d '{
+        "name": "cpu_quota_period", \
+        "value": "1h" \
+     }'
+```
+
+### Get existing global properties
+If no name (type String) is specified, then all existing global properties are fetched. 
+```$xslt
+curl -X GET http://localhost:8080/presto/globalproperty/read/{INSERT_NAME_HERE}
+```
+
+### Update a global property
+Specify all fields in the body, which will overwrite the existing global properties.
```

Comment:
This will overwrite the existing global property's (cpu_quota_period) value only, right? 

Let's say you have 5 global properties, this will overwrite all existing properties and after this operation will only have one property named `cpu_quota_period`?